src/ui/mod.rs

```diff
@@ -1548,6 +1672,20 @@ fn draw_table<B>(
           }
         }
       }
+      TableId::PodcastEpisodes => {
+        if let Some(name_idx) = header.get_index(ColumnId::Title) {
+          if let Some(track_playing_offset_index) =
+            track_playing_index.and_then(|idx| idx.checked_sub(offset))
+          {
+            if i == track_playing_offset_index {
+              formatted_row[name_idx] = format!("▶ {}", &formatted_row[name_idx]);
+              style = Style::default()
+                .fg(app.user_config.theme.active)
+                .add_modifier(Modifier::BOLD);
+            }
+          }
+        }
+      }
       _ => {}
     }
 
```

Comment:
Might be nice to pull in the users podcasts library?

<img width="512" alt="image" src="https://user-images.githubusercontent.com/12150276/94714857-a07ae480-0344-11eb-9a3a-86f4eb96360c.png">

This can be done later though if you'd rather not add any more to this PR - I'll leave that up to you.

Reply:
Yeah I felt like that this PR was getting to be a bit large and I didn't want to overwhelm it with changes. I was planning on cutting a new PR with library podcast support shortly following this merge, if that's alright.